EventListener/ExceptionLogSubscriber.php

```diff
@@ -42,7 +44,7 @@ public function onException($event): void
                 'event' => 'error',
                 'error.kind' => 'Exception',
                 'error.object' => get_class($exception),
-                'message' => $exception->getMessage(),
+                'message' => $exception->getMessage() ? $exception->getMessage() : self::DEFAULT_ERROR_MESSAGE,
```

Comment:
This can be shortened to
```suggestion
                'message' => $exception->getMessage() ?: self::DEFAULT_ERROR_MESSAGE,
```